gamecenter/api/views.py

```diff
@@ -15,6 +18,38 @@
 score_schema = ScoreSchema()
 
 
+def handle_api_key(f):
+    @functools.wraps(f)
+    def decorated_function(*args, **kwargs):
+        if "Authorization" in request.headers:
+            api_key = request.headers["Authorization"][7:]  # from the string form "Bearer <api_key>"
```

Comment:
I did that much lol, but from what little I read on the page I don't get what the "Bearer" part means and how that differs from other auth tokens etc...